common/settings.h

```diff
@@ -32,6 +33,12 @@ using MappingCallback = std::function<std::unique_ptr<fml::Mapping>(void)>;
 using MappingsCallback =
     std::function<std::vector<std::unique_ptr<const fml::Mapping>>(void)>;
 
+using FrameRasterizedCallback =
+    std::function<void(fml::TimePoint build_start,
```

Comment:
Can this just be a `enum Phase` and `fml::TimePoint` pair in the `std::function` args? This will allow us to be add things to this in the future without modifying the signature. Your call.

Reply:
Good idea. Done.